src/installation/guides/chroot.md

```diff
@@ -0,0 +1,222 @@
+# Installation via chroot (x86 or x86_64)
+
+## Prepare Filesystems
+
+These instructions can be followed on any running Linux system that has xbps
+installed, including an already working installation of Void or an official
+[live image](../live-images/prep.md). It is assumed that the host computer is
+connected to the Internet.
```

Comment:
Perhaps list the xbps-static option as well?

Reply:
I don't think it's necessary, but I'm not particularly opposed to it. There should be a more thorough guide on this step that covers this kind of thing (like writing a live image to usb).

Reply:
I think listing the xbps-static option is important because it enables someone who is using another distro to install Void through that. That's why I also consider the ROOTFS way quite relevant (and it only changes the setup in two places).

Reply:
Okay, fair enough. What are the steps necessary? Is there an official binary of xbps-static available? (I can't find it) Is there more documentation on xbps-static than the page currently in the docs?

Reply:
The current page links to this https://alpha.de.repo.voidlinux.org/static/, where you can download a tarball with the compiled software. If you unpack it into a folder and use the xbps-install.static binary (in the alternate root mode), there shouldn't be any issue. It can also be extracted into your `/` directory to "install" xbps, but that feels a bit intrusive. Perhaps this can be added to the Static XBPS page, as an explanation for how to use it. So for now I'd just mention that xbps-static is one way of obtaining xbps, and link to that page. We can then work from that in another PR.

Reply:
Okay, that sounds good. I've added a link to that download page. We can move that link over to the xbps-static docs when there's better documentation there.

I also changed the note that the instructions can be followed on any linux with xbps to any system with xbps that can enter a linux chroot, since these instructions should work also on any BSD that can execute linux binaries.

Reply:
this should be resolved with my next push